### PR TITLE
Fix schema form warnings

### PR DESCRIPTION
**Pull request by kennyt**
px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

File: src/js/components/modals/__tests__/InstallPackageModal-test.js [hunk 1]
```diff
@@ -63,7 +70,7 @@ describe('InstallPackageModal', function () {
         this.instance.getModalContents(),
         this.container
       ));
-      var result = node.querySelectorAll('p')[1];
+      var result = node.querySelectorAll('p')[4];
```
Comment (jfurrow) [inline src/js/components/modals/__tests__/InstallPackageModal-test.js]: Why this change? Just curious.

Comment (kennyt) [inline src/js/components/modals/__tests__/InstallPackageModal-test.js]: this is because we're actually rendering the schema form now, which will itself take up some nodes. therefore we need to look deeper

File: src/js/utils/SchemaFormUtil.js [hunk 2]
```diff
@@ -194,7 +194,8 @@ let SchemaFormUtil = {
 
   validateModelWithSchema(model, schema) {
     let result = tv4.validateMultiple(model, schema);
-    if (result.valid) {
+
+    if (result == null || result.valid) {
```
Comment (kennyt) [inline src/js/utils/SchemaFormUtil.js]: we need to do the `== null` check for tests. it has no significance for the app because tv4 will always return something

File: src/js/components/modals/__tests__/InstallPackageModal-test.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/modals/__tests__/InstallPackageModal-test.js]: Ahh cool. Thanks

**Comment by MatApple** [comment]
I think there are more tests that need fixing too right? I'm seeing `DeploymentsTab-test`, `ServiceDetailTaskTab-test` as well.

**Comment by Poltergeist** [comment]
Looks good lets :ship: this.
